thanosql/parse.py

```diff
@@ -8,16 +9,22 @@ def convert_local_ns(query_string, local_ns) -> str:
         key: val for key, val in local_ns.items() if not isinstance(val, ModuleType)
     }
 
-    # variables need to be converted
-    var_list = list(set(map(str.strip, re.findall(r"=( *?\w+)", query_string))))
+    # find variables need to be converted
+    regex1 = '=( *?\w+)' # regex to find variables coming after "="
+    regex2 = 'FROM( *?\w+)' # regex to find variables coming after "FROM"
+    var_list = list(set(''.join(res).strip(' ') for res in re.findall(f'{regex1}|{regex2}', query_string)))
```

Comment:
```suggestion
    res = re.findall(f'{regex1}|{regex2}', query_string) 
    var_list =set(map(lambda x: x.strip(),  res))
    
    for var in var_list:
      # blah blah
      
    return query_string
```
var_list 너무 길어져서 이렇게 바꿔주시고, 돌아가는지 확인 좀 해주세요.
예전에 코드 왜 저렇게 짰을까요...

그리고 지금 마지막 query_string indent가 하나 밀렸어요.